cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -1832,6 +1833,12 @@ public static String getZKQuorum(CConfiguration cConf) {
     public static final String SERVER_PORT = "jmx.metrics.collector.server.port";
   }
 
+  public static final class KubeMasterEnvironment {
```

Comment:
Shouldn't be defined here. All k8s specific things should stay in the `cdap-kubernetes` module.

Reply:
Removed from here. The reason for adding these constants here was to share property names which need to be set by KubeMasterEnvironment and read by the Main Service that I added. Now I've maintained 2 copies of the property names, one in each file. Can't figure out where to define these constants to make them sharable. If I add them as static strings in KubeMasterEnv, I'll need to add cdap-kubernetes dependency in cdap-master which isn't desirable. Can you suggest a better way than duplicating the strings in both files?

Reply:
Passing kay-value properties with specific prefix defined in MasterEnvironmentContext interface. MainService fetches all properties with the prefix and passes them to JMX Metrics Collector. This way no additional dependencies are required. cdap-master-spi has the prefix constant.